torch/serialization.py

```diff
@@ -851,6 +865,8 @@ def persistent_load(saved_id):
     unpickler.persistent_load = persistent_load
     result = unpickler.load()
 
+    _validate_after_load(result)
```

Comment:
What if the sparse tensor is recursively stored inside of another struct, e.g., a dict?